kuma/search/views.py

```diff
@@ -27,13 +27,23 @@ def search(request, *args, **kwargs):
         return wiki_search(request, *args, **kwargs)
 
     results = search_api(request, *args, **kwargs).data
+
+    # Determine if there were validation errors
+    error = results.get('error') or results.get('q')
+    # Set flag to send results if no errors
+    send_results = True if error is None else False
```

Comment:
This variable is now not needed. You're using it on line 41 and you can just use `error` there. 
I.e. delete this line and change line 41 to `results if not error else None`.